src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -517,15 +517,20 @@ public function handleError($type, $message, $file, $line)
                 $errorAsException ? ['exception' => $errorAsException] : [],
             ];
         } else {
+            if (!$hhvm = \defined('HHVM_VERSION')) {
```

Comment:
Let's remove the variable, it prevents opcache from doing dead code elimination.